node_modules/col-whiteboards/data/whiteboardToPng.js

```diff
@@ -42,15 +42,17 @@ var init = function() {
 
     // Create a PNG stream for these elements
     generatePng(whiteboardElements, function(err, data, dimensions) {
+      // According to https://nodejs.org/api/process.html, "rather than calling process.exit() directly,
```

Comment:
The convention is to use `@see` for external references, so this would become something along the following lines:

```
// Rather than calling process.exit() directly, set the process.exitCode and allow the process to exit naturally
// @see https://nodejs.org/api/process.html
```